app/assets/images/payment_icons/davivienda.svg

```diff
@@ -0,0 +1,29 @@
+<svg
```

Comment:
This applies to all other icons: can you increase the size of the payment brand?

[![alt](https://screenshot.click/23-29-du7u7-yafwt.png)](https://screenshot.click/23-29-du7u7-yafwt.png)

Also can you run the svg through SVGO?

Reply:
Hi @tauthomas01 ! Sorry for the late response. Which size should I apply? I want to finish the process of the icons but I don't undestand what I have to do. Can you help me? Thanks!!

Reply:
Do you have an image editing tool to increase the size? Just increase it a bit so we can see the logo clearer.

Reply:
The size should be as shown in the template, or not?

Reply:
Hi @tauthomas01 , could you check this? Thanks!